app/code/Codifi/Training/ViewModel/Account/CustomerSessionManagementViewModel.php

```diff
@@ -0,0 +1,134 @@
+<?php
+/**
+ * Codifi_Training
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+
+declare(strict_types=1);
+
+namespace Codifi\Training\ViewModel\Account;
+
+use Codifi\Training\Model\ConfigProvider;
+use Codifi\Training\Setup\Patch\Data\AddCustomerAttributeCreditHold;
+use Magento\Customer\Model\Session;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+use Magento\Framework\UrlInterface;
+
+/**
+ * Class CustomerSessionManagementViewModel
+ * @package Codifi\Training\ViewModel\Account
+ */
+class CustomerSessionManagementViewModel implements ArgumentInterface
+{
+    /**
+     * Customer session.
+     *
+     * @var Session
+     */
+    private $session;
+
+    /**
+     * Config provider.
+     *
+     * @var ConfigProvider
+     */
+    private $configProvider;
+
+    /**
+     * Url builder
+     *
+     * @var UrlInterface
+     */
+    private $urlBuilder;
+
+    /**
+     * CustomerSessionManagementViewModel constructor.
+     *
+     * @param ConfigProvider $configProvider
+     * @param Session $session
+     * @param UrlInterface $urlBuilder
+     */
+    public function __construct(
+        ConfigProvider $configProvider,
+        Session $session,
+        UrlInterface $urlBuilder
+    ) {
+        $this->configProvider = $configProvider;
+        $this->session = $session;
+        $this->urlBuilder = $urlBuilder;
+    }
+
+    /**
+     * Get customer attrubute credit hold.
+     *
+     * @return bool
+     * @throws LocalizedException
+     * @throws NoSuchEntityException
+     */
+    public function getCustomerAttrCreditHold(): bool
+    {
+        $customerData = $this->session->getCustomerData();
+        $customerAttribute = $customerData->getCustomAttribute(AddCustomerAttributeCreditHold::ATTRIBUTE_CODE);
+        if ($customerAttribute !== null) {
```

Comment:
Code can be simplified
`return $customerAttribute && $customerAttribute->getValue();`